extensions/amp-access-fewcents/0.1/fewcents-impl.js

```diff
@@ -0,0 +1,184 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {createElementWithAttributes} from '#core/dom';
+import {dict} from '#core/types/object';
+
+import {dev, user} from '#utils/log';
+
+import {
+  CONFIG_BASE_PATH,
+  DEFAULT_MESSAGES,
+  TAG,
+  TAG_SHORTHAND,
+} from './fewcents-constants';
+
+import {CSS} from '../../../build/amp-access-fewcents-0.1.css';
+import {installStylesForDoc} from '../../../src/style-installer';
+
+/**
+ * @implements {../../amp-access/0.1/access-vendor.AccessVendor}
+ */
+export class AmpAccessFewcents {
+  /**
+   * @constructor
+   * @param {!../../amp-access/0.1/amp-access.AccessService} accessService
+   * @param {!../../amp-access/0.1/amp-access-source.AccessSource} accessSource
+   */
+  constructor(accessService, accessSource) {
+    /** @const */
+    this.ampdoc = accessService.ampdoc;
+
+    /** @const @private {!../../amp-access/0.1/amp-access-source.AccessSource} */
+    this.accessSource_ = accessSource;
+
+    /** @private {?Node} */
+    this.innerContainer_ = null;
+
+    /** @private {?Node} Main container where paywall will be rendered */
+    this.dialogContainer_ = null;
+
+    /** @const @private {JsonObject}  Stores the config passed by the publisher */
+    this.fewcentsConfig_ = this.accessSource_.getAdapterConfig();
+
+    /** @private {string} Authorize endpoint to get paywall data*/
+    this.authorizeUrl_ = this.prepareAuthorizeUrl_();
+
+    /** @private {!JsonObject} */
+    this.i18n_ = Object.assign(dict(), DEFAULT_MESSAGES);
+
+    // Install styles.
```

Comment:
nit: unnecessary comment for self documenting function name.

Reply:
Changes done.